docs/source/user/installing.rst

```diff
@@ -0,0 +1,58 @@
+
+Getting started
+===============
+
+Installing using Conda
+----------------------
+
+The easiest way to install numba and get updates is by using Conda,
+a cross-platform package manager and software distribution maintained
+by Continuum Analytics.  You can either use `Anaconda
+<http://continuum.io/downloads.html>`_ to get the full stack in one download,
+or `Miniconda <http://conda.pydata.org/miniconda.html>`_ which will install
+the minimum packages needed to get started.
+
+If/once you have conda installed, just type::
+
+   $ conda install numba
+
+or::
+
+   $ conda update numba
+
+Installing from source
+----------------------
+
+We won't cover requirements in detail here, but you can get the bleeding-edge
+source code from `Github <https://github.com/numba/numba>`_::
+
+   $ git clone git://github.com/numba/numba.git
+
+Source archives of the latest release can be found on
+`PyPI <https://pypi.python.org/pypi/numba/>`_.
+
+You will need a C compiler corresponding to your Python installation, as
+well as the `Numpy <http://www.numpy.org/>`_ and
+`llvmlite <https://github.com/numba/llvmlite>`_ packages.  See :ref:`buildenv`
+for more information.
+
+Checking your installation
```

Comment:
Is it worth also mentioning `numba.test()` here? For example, this might pick up a problem with the CUDA setup even if Numba imports OK.